### PR TITLE
backend, new calendar and fix for organization data

### PR DESCRIPTION
**Pull request by Szanki**

### CODE REVIEW

File: client/src/components/calendar/CalendarDemo.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+							dayMaxEvents
+							weekends={weekendsVisible}
+							select={handleDateSelect}
+							eventContent={renderEventContent} // custom render function
+							eventClick={handleEventClick}
+							eventsSet={(events: EventApi[]) => setCurrentEvents(events)} // called after events are initialized/added/changed/removed
+							eventAdd={(e) => {
+								addEvent({ variables: { event: e.event }, context: { event: e.event } });
+							}}
+							eventChange={(e) => setTimeout(() => e.revert(), 3000)}
```
Comment (peleek) [inline client/src/components/calendar/CalendarDemo.tsx:61]: Please remove that revert functions, they should be replaced with API calls in near future

File: client/src/components/calendar/calendarStyle.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+.demo-app-sidebar-section {
+	padding: 2em;
+}
+
+.demo-app-main {
+	flex-grow: 1;
+	padding: 3em;
+}
+
+.fc {
```
Comment (peleek) [inline client/src/components/calendar/calendarStyle.css:53]: fc is not meaningful. Anyway it's not used anywhere

File: client/src/components/calendar/calendarStyle.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
+html,
+body,
+body > div {
```
Comment (peleek) [inline client/src/components/calendar/calendarStyle.css:3]: Why are there  some global styles?

File: client/src/components/calendar/calendarStyle.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
+html,
```
Comment (peleek) [inline client/src/components/calendar/calendarStyle.css:1]: Comments regarding this whole file:
- demo-app-* classes are not used anywhwere
- We should keep our convention to write styles in jss

File: client/src/components/calendar/types.ts [hunk 5]
```diff
@@ -1,5 +1,20 @@
+import { EventApi, DateSelectArg, EventClickArg, EventContentArg } from '@fullcalendar/react';
+
 export type Event = {
 	start: number;
 	end: number;
 	title: string;
 };
```
Comment (peleek) [inline client/src/components/calendar/types.ts:7]: add empty line

File: client/src/components/userSettings/ChangePassword.tsx [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
-import { USER_PASSWORD_MUTATION } from './queries/changeUserPassword';
+import { Grid, Typography, Box, Button } from '@material-ui/core';
 import MuiAlert, { AlertProps } from '@material-ui/lab/Alert';
 import Snackbar from '@material-ui/core/Snackbar';
 import { useMutation } from '@apollo/react-hooks';
+import { USER_PASSWORD_MUTATION } from './queries/changeUserPassword';
+import { userSetingsStyles } from './userSettings.style';
+import { ChangePasswordField } from './ChangePasswordField';
 
 const Alert = (props: AlertProps) => {
```
Comment (peleek) [inline client/src/components/userSettings/ChangePassword.tsx:10]: Move it to new  file

File: client/src/context/calendarContext.tsx [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+	}
+};
+
+export const CalendarContextProvider = (props) => {
+	const [{ events }, dispatch] = useReducer(calendarReducer, intialCalendarState);
+
+	const addEvent = (event) => {
+		dispatch({ type: 'ADD_EVENT', payload: event });
+	};
+	const RemoveEvent = (id) => {
```
Comment (peleek) [inline client/src/context/calendarContext.tsx:32]: cammelCase instead of PascalCase

File: client/src/pages/CalendarPage.tsx [hunk 8]
```diff
@@ -1,16 +1,76 @@
...
-				<CalendarPalette />
+				<CalendarSidebar
+					setWeekendsVisible={setWeekendsVisible}
+					weekendsVisible={weekendsVisible}
+					currentEvents={currentEvents}
+				/>
 			</Grid>
 			<Grid item xs={10}>
-				<Calendar />
+				{/* <Calendar /> */}
```
Comment (peleek) [inline client/src/pages/CalendarPage.tsx:64]: Remove commented code

File: client/src/components/calendar/calendarStyle.css (hunk 2 above)
Comment (Szanki) [reply client/src/components/calendar/calendarStyle.css:53]: it's used by the library without our knowledge (is not in the code)

**Pull request merged**